src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -75,6 +76,10 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
                 {
                     // TODO: We need to handle format parameter.
                 }
+                else if (string.IsNullOrWhiteSpace(query.Item1))
```

Comment:
Can we just add this to the unsupported search parameters instead of throw exception?